src/database/queries/opportunities/selectById.sql

```diff
@@ -2,9 +2,17 @@ SELECT opportunity_to_json(opportunities.*) AS object
 FROM opportunities
 WHERE
 	opportunities.id = :opportunityId
-	AND has_funder_permission(
-		:authContextKeycloakUserId,
-		:authContextIsAdministrator,
-		opportunities.funder_short_code,
-		'view'
+	AND (
+		has_funder_permission(
+			:authContextKeycloakUserId,
+			:authContextIsAdministrator,
+			opportunities.funder_short_code,
+			'view'
+		)
+		OR has_opportunity_permission(
```

Comment:
If we view the opportunity as being "under" a funder, I think this should would be an `AND`. But maybe we want to keep them separate, you could view the opportunity without seeing funder info, thus `OR`. Hmm.

Reply:
This is explicitly an or -- the whole point of these permissions is that users who are not members of the funder organization need to be able to make proposals.